ogr/services/base.py

```diff
@@ -131,6 +92,55 @@ def search_in_pr(
 
         return search_in_comments(comments=all_comments, filter_regex=filter_regex)
 
+    @deprecated("since 0.9.0, will be removed in 1.4.0: Use methods on Issue objects")
+    def get_issue_comments(
+        self,
+        issue_id,
+        filter_regex: str = None,
+        reverse: bool = False,
+        author: str = None,
+    ) -> List[IssueComment]:
+        return self.get_issue(issue_id).get_comments(filter_regex, reverse, author)
+
+    @deprecated("since 0.9.0, will be removed in 1.4.0: Use methods on Issue objects")
```

Comment:
Not sure, if the version `1.0.0` is the ancestor of the `0.9.0`.

Something like this:

```suggestion
    @deprecated("since 0.9.0, will be removed in 0.14.0 (or 1.0.0 if it comes sooner): Use methods on Issue objects")
```